operators/hpe-csi-operator/destinations/certified-operators/current-version/manifests/hpe-csi-operator.clusterserviceversion.yaml

```diff
@@ -30,31 +50,57 @@ metadata:
+              "csiNodeDriver": "quay.io/hpestorage/csi-driver@sha256:d96e2a63029267abcb3eaf032eb70401861866d636bdb6f192ceb6cee954bb67",
+              "csiNodeDriverRegistrar": "registry.k8s.io/sig-storage/csi-node-driver-registrar@sha256:f25af73ee708ff9c82595ae99493cdef9295bd96953366cddf36305f82555dac",
+              "csiProvisioner": "registry.k8s.io/sig-storage/csi-provisioner@sha256:405a14e1aa702f7ea133cea459e8395fe40a6125c088c55569e696d48e1bd385",
+              "csiResizer": "registry.k8s.io/sig-storage/csi-resizer@sha256:a541e6cc2d8b011bb21b1d4ffec6b090e85270cce6276ee302d86153eec0af43",
+              "csiSnapshotter": "registry.k8s.io/sig-storage/csi-snapshotter@sha256:2e04046334baf9be425bb0fa1d04c2d1720d770825eedbdbcdb10d430da4ad8c",
+              "csiVolumeGroupProvisioner": "quay.io/hpestorage/volume-group-provisioner@sha256:51a49a6848e7c22c4c4307496724c6260497aa93d70858b44c16972e12fd18f1",
+              "csiVolumeGroupSnapshotter": "quay.io/hpestorage/volume-group-snapshotter@sha256:39914c29b33dcb11b5df8d15b5c90913bcf1c994b44d7f202b8d1cb52e9919a9",
+              "csiVolumeMutator": "quay.io/hpestorage/volume-mutator@sha256:57a4449afc2c3e31586e6fbb760f6e2503abcaa1baca70c839e039ea66534135",
+              "nfsProvisioner": "quay.io/hpestorage/nfs-provisioner@sha256:c15b5e76c58cb3f75f81d5e60f61baddf891fa8c49e2f6f580226f129413cd07",
+              "nimbleCSP": "quay.io/hpestorage/alletra-6000-and-nimble-csp@sha256:ab4003f93032129e7f5768dd9b2b047fbfd435b8e0f03e4a60d6d30693320758",
+              "primera3parCSP": "quay.io/hpestorage/alletra-9000-primera-and-3par-csp@sha256:320c335062338378889c1dde2ecba955d3c4d6f4fab120ea075bc4a6c5ffe7af"
+            },
             "iscsi": {
-              "chapPassword": "",
-              "chapUser": ""
+              "chapSecretName": ""
             },
-            "kubeletRootDir": "/var/lib/kubelet/",
+            "kubeletRootDir": "/var/lib/kubelet",
             "logLevel": "info",
             "node": {
               "affinity": {},
               "labels": {},
               "nodeSelector": {},
+              "resources": {
+                "limits": {
+                  "cpu": "2000m",
+                  "memory": "1Gi"
+                },
+                "requests": {
+                  "cpu": "100m",
+                  "memory": "128Mi"
+                }
+              },
               "tolerations": []
-            },
-            "registry": "quay.io"
+            }
           }
         }
       ]
     capabilities: Basic Install
     categories: Storage
     certified: "true"
-    containerImage: registry.connect.redhat.com/hpestorage/csi-driver-operator@sha256:4c8b1da88cff244bd2d2284249e4247f7d2a1a2fb765e988c3017d81e3dc4d29
-    createdAt: "2024-04-09T19:42:44Z"
+    containerImage: registry.connect.redhat.com/hpestorage/csi-driver-operator@sha256:891a287c7a0fd84e12f4998c33bd072283d0d7d975a2cb25d8b45025cfdd1fb7
```

Comment:
This digest can be updated to the latest.

sha256:188c4ec476c1ba80403653bf7663a1c79aba2714df56d63909963a073f14b392

Reply:
Submit a separate PR once the upstream image has been published.